hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -502,27 +537,10 @@ public void initializeStatesApi(
                         .activeProfile(),
                 trigger,
                 deserializedVersion == null ? "<NONE>" : deserializedVersion);
-
-        logger.info("Initializing Hedera platform state");
-        final var platformStateMigrations = serviceMigrator.doMigrations(
-                state,
-                servicesRegistry.subRegistryFor(EntityIdService.NAME, PlatformStateService.NAME, RosterStateId.NAME),
-                deserializedVersion == null ? null : new ServicesSoftwareVersion(deserializedVersion),
-                version,
-                bootstrapConfigProvider.getConfiguration(),
-                platformConfiguration,
-                UNAVAILABLE_NETWORK_INFO,
-                UNAVAILABLE_METRICS);
-        migrationStateChanges = new ArrayList<>();
-        migrationStateChanges.addAll(platformStateMigrations);
```

Comment:
Seems like this is dead code, thats why it was removed?

Reply:
This was shadowing an instance field with the same name; so just used the field.